sqlstore/src/main/resources/db/migration/V8801__library_size.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE Library ADD COLUMN `size` bigint(20);
+
+UPDATE Library SET size = (SELECT insertSize FROM LibraryQC WHERE LibraryQC.library_libraryId = Library.libraryId ORDER BY qcDate LIMIT 1);
```

Comment:
The insert sizes are preserved in the migration. This is just picking “the best” one and it can be changed.